force_gromacs/pipeline/tests/test_gromacs_pipeline.py

```diff
@@ -51,13 +64,29 @@ def test_named_steps(self):
             list(self.pipeline.named_steps.keys())
         )
 
+        self.assertIsInstance(
+            self.pipeline.named_steps['file_tree'], GromacsFileTreeBuilder
```

Comment:
I would love to know more about this feature: `.named_steps[Str]`, and how is it different from just calling `[Str]`.

Reply:
You could consider it a different use case as we are not relying on the functionalities of our overloaded `__getitem__` method.